doc/feature/pointDataEvolution.md

```diff
@@ -0,0 +1,85 @@
+Evolution of data at a point over time
+======================================
+
+One of the features that would make the tool extremely useful for scientists, is
+the ability to obtain the evolution of the data at a specific point over time.
+For example, they could load a current report and compute the LFP for a specific
+target, and once they have their full volume, they could focus on a specific
+point of the scene (e.g. a voxel), and obtain a 2D curve showing the evolution
+of voltage at that point over time.
+
+## Requirements
+
+* Users should be able to specify the coordinates of a 3D point as input.
+* As output, users will obtain a series of values corresponding to the evolution
+  of the data at that specific point over time (e.g. one value per simulation
+  frame).
+* The computation should be independent from the voxelization step, as the
+  point for which the values will be computed should not be limited to discrete
+  voxels, but a continuous 3D space.
+* Optionally, it could use ZeroEQ to stream the data to an external component,
+  like a GUI showing a 2D graph that is updated dynamically as the values are
+  being computed.
```

Comment:
I wouldn't mention anything to be considered optional at this stage. Also note that streaming poses some specific problems on its own as the spike and compartment report discussion have shown.